### PR TITLE
Final removal of Private Kit (leaving only attributions)

### PR DESCRIPTION
**Pull request by kenpugsley**
Final removal of Private Kit from the repo, except for the attributions.  Should set us on the path to be able to produce final app store release builds.

I've also included the rewording on the location/accelerometer permission description (to take out the refugee wording).

What I've tested (on Mac ... not Linux - sorry @penrods 

- pulled branch fresh the repo
- yarn install, yarn install:pod
- running on both Android and iOS simulators
- building and installing on iPhone from xcode
- building Android .apk from command line and sideloading on Android phone

All seems to work.

### CODE REVIEW

File: android/.settings/org.eclipse.buildship.core.prefs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+arguments=
```
Comment (tstirrat) [inline android/.settings/org.eclipse.buildship.core.prefs:1]: .gitignore these elipse specific file additions

File: android/.project [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tstirrat) [inline android/.project:1]: .gitignore eclipse stuff

File: android/.settings/org.eclipse.buildship.core.prefs (hunk 1 above)
Comment (kenpugsley) [reply android/.settings/org.eclipse.buildship.core.prefs:1]: Hmmm... not sure where this came from.  Will remove.

File: android/.project (hunk 2 above)
Comment (kenpugsley) [reply android/.project:1]: same as above ... not sure where this came from

**Pull request merged**